app/uk/gov/hmrc/exports/models/declaration/notifications/Notification.scala

```diff
@@ -16,20 +16,36 @@
 
 package uk.gov.hmrc.exports.models.declaration.notifications
 
-import java.time.LocalDateTime
+import java.time.{Instant, LocalDateTime, ZoneOffset}
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Json, Reads, _}
 import uk.gov.hmrc.exports.models.declaration.submissions.SubmissionStatus.SubmissionStatus
 
 case class Notification(
   actionId: String,
   mrn: String,
-  dateTimeIssued: LocalDateTime,
+  dateTimeIssued: Instant,
   status: SubmissionStatus,
   errors: Seq[NotificationError],
   payload: String
 )
 
 object Notification {
-  implicit val format: OFormat[Notification] = Json.format[Notification]
+  implicit val readLocalDateTimeFromString: Reads[Instant] = implicitly[Reads[LocalDateTime]]
+    .map(_.atZone(ZoneOffset.UTC).toInstant)
```

Comment:
Are we safe assuming the incoming dateTime is UTC?   If it has a timezone component would it not be better to read as Instant? (I.e. use the zone from the incoming data)

Reply:
This would be the values we have on our database, which are in 

> LocalDateTime

 format.
We are fine here